requirements.txt

```diff
@@ -4,7 +4,22 @@ sally (https://github.com/rieck/sally)
 R
 mergecap
 pygraphviz
+progressBar
+numpy
 python-Levenshtein
 
 # Optional
 cuckoo (https://github.com/cuckoobox/cuckoo)
+
+# The derrick, Sallary tools need to be installed one-by-one
+# On Centos, R, mergecap, pygraphviz and python-Levenshtein can be installed by;
```

Comment:
is it really necessary to provide instructions to install standard packages for each specific platform?